acmw-db-app/pages/api/meeting/[pid].js

```diff
@@ -0,0 +1,38 @@
+import pgQuery from '../../../postgres/pg-query.js';
+
+// GET /api/meeting/averageAttendance
+async function averageAttendance () {
+    var attendees = 0;
+    const meetingIds = await pgQuery(`SELECT id FROM meeting`);
+    for (var i = 0; i < meetingIds.rows.length; i++) {
```

Comment:
while the meeting_student table is theoretically cleared out after every academic year, i would prefer to explicitly check that the meetings you are counting are within the given academic year by checking the semester column values